src/components/header/Header.tsx

```diff
@@ -90,15 +93,53 @@ const Header = () => {
       console.log(modalData);
     });
   };
-  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
-  const openMenu = anchorEl;
+  const [menuAnchorEl, setMenuAnchorEl] = React.useState<null | HTMLElement>(
+    null
+  );
+
+  const openMenu = menuAnchorEl;
   const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
-    setAnchorEl(event.currentTarget);
+    setMenuAnchorEl(event.currentTarget);
   };
+
   const handleClose = () => {
-    setAnchorEl(null);
+    setMenuAnchorEl(null);
   };
 
+  const BASE_URL = "https://www.dealbuddy.co.nz/api";
```

Comment:
instead of hard code url make config for such data. Use can also use `dotenv` package to manage such environment base config.

Reply:
Sir this "BASE_URL" not needed now so i will remove it from here.